asserts/ifacedecls.go

```diff
@@ -143,7 +145,7 @@ type regexpAttrMatcher struct {
 }
 
 func compileRegexpAttrMatcher(cc compileContext, s string) (attrMatcher, error) {
-	rx, err := regexp.Compile("^" + s + "$")
+	rx, err := regexp.Compile("^(" + s + ")$")
```

Comment:
This is a nice catch. It means that now 'foobar|z' correctly matches 'foobar' and 'z' and that you need to use 'fooba(r|z)' to match 'foobar' and 'foobaz'.